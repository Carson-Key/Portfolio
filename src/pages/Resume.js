// Components
import Container from '../components/Container'

const Resume = () => {
    return (
		<Container>
			<p>Resume</p>
		</Container>
	)
}

export default Resume
