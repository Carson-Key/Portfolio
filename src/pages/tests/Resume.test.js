// Packages
import renderer from 'react-test-renderer'
// Pages
import Resume from '../Resume'

describe('Resume Page', () => {
    it('Should Render Resume page', () => {
        const resume = renderer.create(
            <Resume />
        ).toJSON()
        expect(resume).toMatchSnapshot()
    })
})