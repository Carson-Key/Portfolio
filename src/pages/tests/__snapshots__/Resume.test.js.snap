// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume Page Should Render Resume page 1`] = `
<p>
  Resume
</p>
`;
